Sources/Fuzzilli/Execution/REPRL.swift

```diff
@@ -49,6 +49,14 @@ public class REPRL: ComponentBase, ScriptRunner {
         self.maxExecsBeforeRespawn = maxExecsBeforeRespawn
         super.init(name: "REPRL")
 
+        #if os(macOS)
```

Comment:
This is a bit of an unfortunate workaround. If there's no way to avoid this with JSC build options, then I think it should go into the [JSCProfile](https://github.com/googleprojectzero/fuzzilli/blob/main/Sources/FuzzilliCli/Profiles/JSCProfile.swift)

Reply:
Yes, truly it should .
(With the next problem, I find